package.json

```diff
@@ -141,6 +141,7 @@
     "eslint-plugin-jsdoc": "^27.0.3",
     "eslint-plugin-prettier": "^3.1.3",
     "globby": "^11.0.2",
+    "got": "^11.8.2",
```

Comment:
Need a library to do REST calls, `got` seems very popular so I grabbed it. Might be a suitable replacement for `request`.

Reply:
I just saw that someone pulled in `got` to plugin-release-management, so not a new discovery ;).